test/ably/resttoken_test.py

```diff
@@ -179,6 +180,12 @@ def test_toke_details_from_json(self):
             TokenDetails.from_json(token_details_str),
         )
 
+    # Issue #71
+    @dont_vary_protocol
+    def test_request_token_float(self):
+        lifetime = datetime.timedelta(hours=4)
+        self.ably.auth.request_token({'ttl': lifetime.total_seconds() * 1000})
```

Comment:
> If you want I could make a different PR after this one, where a timedelta object is accepted for Duration values, and a datetime object is accepted for Time values. Don't know if changing how an integer is interpreted (ms or s) is worth it, as it would break backwards compatibility.

Yes, I agree, it's a bit messy now and it should have been ms if that is idiomatic, especially as we use ms by default in Ably unless the client lib requires otherwise.

One option could be to treat any number < 1000 as seconds, but in future that could really backfire.  Instead, I prefer your appraoch, it's typed as a Duration / int (representing ms or seconds depending on the documentation).  Can you go ahead and do that please?